.mailmap

```diff
@@ -151,4 +151,4 @@ Jeffrey Aaron Paul <jeffrey.paul2000@gmail.com> Jeffrey Aaron Paul <50923538+jef
 Jeffrey Aaron Paul <jeffrey.paul2000@gmail.com> jeffreypaul15 <jeffrey.paul2000@gmail.com>
 Tathagata Paul <tathagatapaul7@gmail.com> 4molybdenum2 <tathagatapaul7@gmail.com>
 SophieLemos <sophielemos@outlook.com> sophielemos <sophielemos@outlook.com>
-Kritika Ranjan <kritikaranjan28@gmail.com> Kritika <kritikaranjan28@gmail.com>
+Kritika Ranjan <kritikaranjan28@gmail.com> Kritika <kritikaranjan28@gmail.com>
```

Comment:
I think you might want to undo this change.

Reply:
yeah, I'll do that, sorry for the inconvenience